### ISSUE TITLE
[LSP] META-INF/services/org.eclipse.xtext.ISetup overridden when using multiple languages

### ISSUE THREAD

**Issue by andreasmuelder** [eclipse/xtext-core#1684]
When using more than one language and bundling the plugins as a fat jar, the maven-shade-plugin overrides the contents of 'org.eclipse.xtext.ISetup' an thus only one language service is started properly. This can be fixed by adding the ServicesResourceTransformer to the pom.xml:
```
<transformer 
  implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
</transformers>
```

**Comment by cdietrich** [comment]
Gradle is also affected. with provide a PR for the gradle part

**Comment by cdietrich** [comment]
fixed in 2.26
